src/redux/actions/componentConfig/getTrajectoryPlotPseudoTime.js

```diff
@@ -15,7 +14,6 @@ const getTrajectoryPlotPseudoTime = (
   // Currenty monocle3 only trajectory analysis only supports
   // UMAP embedding. Therefore, this embedding is specifically fetched.
   const embeddingMethod = 'umap';
-  const embeddingETag = await getEmbeddingETag(experimentId, getState, dispatch, embeddingMethod);
```

Comment:
This is being used by the worker to get the embedding if it exists in work results. I checked briefly in the api and it doesn't seem like the ETag is being added anywhere. Does trajectory analysis still work?

Reply:
fixed in the api, I don't want the UI to deal with any ETag generation

Reply:
Ok, so with the fix implemented in the api now we have most of the body of the request generated in the UI and then the api adds another parameter `ETag`, right?

I feel like this might be pretty easy to break unknowingly in the future, but not sure what would be a good alternative honestly.

Perhaps at the very least we should add comments to the UI trajectory and processed rds body generation code that explain clearly this isn't the full body because the ETag is added in the api.

Reply:
it's not most of the body, it is a single parameter. I left it a bit hacky because I think the way the worker is implemented in this is what should be changed to stop using this way of adding precomputed embeddings. There were some iniatives on that regard like making the rds mutable, or eventually a possible move into scanpy. I am not in favor of adding comments here because when we change the worker to stop doing that, we just have to remove those lines from the API and I'm pretty sure we'll forget to remove the comments from the UI because there's no code there related

Reply:
> it's not most of the body, it is a single parameter

ok, the point was that the body is split up

> the way the worker is implemented in this is what should be changed to stop using this way of adding precomputed embeddings

agreed, this is partly the reason I proposed computing the embedding in the pipeline and storing in rds